db/migrate/20230223000336_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reviews do |t|
+      t.text :comment
+      t.float :rating
+      t.references :art, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
```

Comment:
You say that :rating is a float but in the validation you have a validation that it can only be an integer.
You need to change this to `t.integer :rating`